### ISSUE TITLE
Not showing errors when `RUN` is used in `ONBUILD`

### ISSUE THREAD

**Issue by matrixik** [hadolint/hadolint#217]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
Show errors for all commands.

### Actual behavior
Hadolint is showing error about not pinned version installation in `apk add` in simple `RUN` but is not seeing same problem in `ONBUILD RUN`.

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

`Haskell Dockerfile Linter v1.7.1-0-gfcaddad`

```bash
λ hadolint Dockerfile
Dockerfile:6 DL3018 Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`
```

**Dockerfile** (if relevant)

```Dockerfile
FROM scratch

ONBUILD RUN \
            apk add --no-cache git

RUN \
    apk add --no-cache git
```

**Additional environment details (OS, stack version, etc.)**
```
System:    Host: zybortd Kernel: 4.10.0-42-generic x86_64 (64 bit gcc: 6.3.0)
           Desktop: Gnome  (Gtk 3.22.11-0ubuntu3) Distro: Ubuntu 17.04
```

**Comment by lorenzo** [comment]
Fun, that is actually fixed in my latest PR. Should be released today!

**Comment by lorenzo** [comment]
Oops, no... I was mistaken. It is not fixed by it, I just made it easier to fix. Will look into it

**Comment by matrixik** [comment]
I can confirm it's working. Thank you.
